web/panic_capture.go

```diff
@@ -0,0 +1,30 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/edaniels/golog"
+)
+
+// PanicCapture allows recovery during a request handler from panics. It prints a
+// formatted log to the underlying logger.
+type PanicCapture struct {
+	Logger golog.Logger
+}
+
+// Recover captures and prints the error if recover() has an error.
+func (p *PanicCapture) Recover(w http.ResponseWriter, r *http.Request) {
```

Comment:
optional: our middleware can still be catching panics and we could repanick here unless you think it's right to just squash it here

Reply:
It's an option. The http server panic recovering logic would kick in a fire off another log. I don't see much value in it.

Reply:
okay cool. as is is fine then!